src/questions/venus.yaml

```diff
@@ -0,0 +1,229 @@
+      
+    - content: XVS is the governance token of Venus
+      key: B
+      
+    - content: VAI is the governance token of Venus
+      key: C
+      
+    - content: VNS is the governance token of Venus
+      key: D
+            
+- uuid: 6d8ee145-b7aa-463b-8cd6-842a10aafb75
+  type: SingleChoice
+  content: How is the amount of XVS distrubuted decided?
+  hint: NoHint
+  explanation: How much XVS is distributed to suppliers and borrowers in each market is determined by “venus speed”. This number is updated regularly to ensure that XVS is being distributed proportionally to the amount of utility each market provides.
+  answerKeys:
+    - A
+  difficultyLevel: "Medium"
+  choices:
+    - content: Decided by a value called "venus speed"
+      key: A
+      
+    - content: Decided by a value called "XVS distrubution rate"
+      key: B
+      
+    - content: Decided by VenusDAO
+      key: C
+      
+    - content: Decided by price oracles
+      key: D
+
+- uuid: aee45a32-dcee-48b3-a42e-4e545b52fdf7
+  type: SingleChoice
+  content: What is VRT? What is its use case?
+  hint: NoHint
+  explanation: VRT is Venus Reward Token, it can be redeemed for XVS at a fixed rate.
+  answerKeys:
+    - A
+  difficultyLevel: "Medium"
+  choices:
+    - content: VRT is Venus Reward Token, used to redeem XVS at a fixed rate
+      key: A
+      
+    - content: VRT is Venus Repeat Token, used to borrow XVS at a fixed rate
+      key: B
+      
+    - content: VRT is Venus Reward Token, used to gain interest at a fixed rate
+      key: C
+      
+    - content: VRT is Venus Return Token, used to redeem XVS at a fixed rate
+      key: D
+      
+
+
+- uuid: 87d32f53-099d-4188-89a1-e942d7d408e3
+  type: SingleChoice
+  content: There is no vault mechanism in Venus
+  hint: NoHint
+  explanation: Venus Vault is a mechanism that locks XVS to improve the system’s anti-risk ability and distributes staking rewards.           
+  answerKeys:
+    - B
+  difficultyLevel: "Low"
+  choices:
+    - content: "True"
+      key: A
+      
+    - content: "False"
+      key: B
+      
+- uuid: 4b108797-6c8a-4235-8675-90174da10dc5
+  type: SingleChoice
+  content: How do vaults help Venus in the case of a shortfall (negative account liquidity) event?
+  hint: NoHint
+  explanation: After a risk event occurs in Venus and only if the Venus risk fund account is not sufficient to cover the shortfall, the XVS depositor in the vault will pay a maximum of 10% of the total deposit amount to compensate for the loss.
+  answerKeys:
+    - B
+  difficultyLevel: "High"
+  choices:
+    - content: There are no vaults in Venus
+      key: A
+      
+    - content: In case Venus risk funds are not sufficient to cover the shortfall, the XVS vault will be liquidated to to a maximum of 10% per user's deposited XVS to cover the loss
+      key: B
+      
+    - content: Immediately, the XVS vault will be liquidated to to a maximum of 10% per user's deposited XVS to cover the loss
```

Comment:
Is C a wrong option?

Reply:
yes, only when the risk funds are not sufficient to cover the losses, the xvs vault will be liquidated. Not immediately when a shortfall occurs